generators/client/templates/angular/src/main/webapp/_index.html

```diff
@@ -41,6 +43,11 @@
             script.type = 'text/javascript';
             document.getElementsByTagName('head')[0].appendChild(script)
         }
+        /*if ('serviceWorker' in navigator) {
```

Comment:
may be put this in a separate <script> block and comment that like how we do for GA below